src/test/java/org/fcrepo/camel/integration/FcrepoTransformIT.java

```diff
@@ -65,8 +65,7 @@ public void testTransform() throws InterruptedException {
 
         final String ldpath = "@prefix fcrepo : <http://fedora.info/definitions/v4/repository#>\n" +
             "id      = . :: xsd:string ;\n" +
-            "title = dc:title :: xsd:string;\n" +
-            "uuid = fcrepo:uuid :: xsd:string;";
```

Comment:
If you are removing the uuid field, you can also remove the `fcrepo` prefix above